### ISSUE TITLE
 Google Maps navigation aborts every time screen is off

### ISSUE THREAD

**Issue by kwinz** [GrapheneOS/os-issue-tracker#3430]
Google Maps navigation aborts every time screen is off.
Not sure if this is a regression.

Pixel 7 Pro
GrapheneOS Build 2024040300
com.google.android.maps versionCode 1067455164

**Comment by Pietro395** [comment]
Tried it and it doesn't happen to me on P6a

Build 2024040900

**Comment by kwinz** [comment]
No, it did not. I can still reliably reproduce the error after the update and a reboot.

@thestinger It only happens for bike navigation though, not for car navigation, walking or public transport naviation. Very strange.

**Comment by kwinz** [comment]
Good idea, however clearing app data did not fix the bug. Still bike navigations abort on screen off.

**Comment by Pietro395** [comment]
> No, it did not. I can still reliably reproduce the error after the update and a reboot.
> 
> @thestinger @Pietro395 It only happens for bike navigation though, not for car navigation, walking or public transport naviation. Very strange.
> 
> I agree, it looks like an app issue! Thanks!

I confirm that it happens with the bike mode

**Comment by kwinz** [comment]
I have reported this via the app's "provide product feedback" - "Report an issue" fuctionality. Not sure if there is anything else I we can do.
